dev/Soniyanayak51/classificationModelFunctions.py

```diff
@@ -0,0 +1,175 @@
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import tensorflow as tf
+import sklearn as skl
+from sklearn.model_selection import train_test_split
+import matplotlib.pyplot as plt
+from sklearn.utils import shuffle
+from sklearn.metrics import confusion_matrix
+import seaborn as sns
+import matplotlib.gridspec as gridspec
+from sklearn.preprocessing import StandardScaler
+from sklearn.manifold import TSNE
+from pandas import set_option
+from sklearn.svm import SVC
+from sklearn.linear_model import LogisticRegression
+from sklearn.datasets import load_iris
+
+
+def loadDataAndPrintHead():
+    '''
+    Loads data using pandas read_csv.
+    '''
+    df = pd.read_csv("/home/soniya51/Downloads/default of credit card clients.csv")
```

Comment:
please split out the local filepath as a variable to make local review easier.

Reply:
@mlopatka Thanks for the review! 
Sure, I'll incorporate the suggested changes.

Reply:
Also, please make sure to use the CSV file included in the repo. That way anyone who runs the code should get consistent results. This will require you change other things in the notebook that depend on the CSV structure, eg column headings.